lib/prerequisite.js

```diff
@@ -40,6 +40,18 @@ module.exports = (input, pkg, opts) => {
 			})
 		},
 		{
+			title: 'Check yarn.lock integrity',
+			enabled: () => opts.yarn,
+			task: () => execa.stdout('yarn', ['check', '--integrity'])
+				.catch(err => {
+					if (err.code === 'ENOENT') {
```

Comment:
On Windows, `execa` has an `ENOENT` code on failure too (not just if something cannot be found). I just tried bumping with yarn installed and got `Yarn not available...` the error.